ui/src/components/IndividualEntry.vue

```diff
@@ -223,10 +225,16 @@ export default {
   },
   methods: {
     selectEntry() {
-      if (this.isLocked) {
-        return;
+      if (!this.isLocked && !this.timeout) {
+        this.timeout = window.setTimeout(() => {
+          this.timeout = null;
+          this.$emit("select");
+        }, 350);
```

Comment:
I guess this is the timeone, isn't it? If that's the case, it would be better to have it under a constant and not like a magic number: https://en.wikipedia.org/wiki/Magic_number_(programming).

Reply:
You're right. I fixed it.